services/QuillLMS/spec/services/formatter_spec.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Formatter do
+  subject { described_class.run(selected_format, value) }
+
+  context 'default' do
+    let(:selected_format) { described_class::DEFAULT }
+    let(:value) { 'Should not be modified' }
+
+    it { expect(subject).to eq(value) }
```

Comment:
The spec file switches between `it { expect(subject)... }` and `it { is_expected }`. For consistency, what about using `it { is_expected... }` throughout?